### DISCUSSION TITLE
Postinstall Causing Issues

### DISCUSSION THREAD

**Discussion by imsk17** [evilmartians/lefthook#652] [Q&A]
Hi, So I have a nodejs typescript library package that has a post-install script "lefthook install".  Now, I build this library on the same repo into a seprate branch using github actions. and when i import this library into any other project, that project requires me to install lefthook. Any workaround for this?

**Comment by mrexox** [comment]
Hey! What is the name of this package?

Usually lefthook doesn't require explicit postinstall scripts because it does install automatically if the `CI` env variable is not set. So, in the CI lefthook doesn't install by default. Of course if `lefthook install` is not called explicitly
